src/Tasks/Microsoft.Common.CrossTargeting.targets

```diff
@@ -46,6 +53,18 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </ItemGroup>
   </Target>
 
+  <Target Name="GetTargetFrameworksWithPlatformFromInnerBuilds"
+          DependsOnTargets="_ComputeTargetFrameworkItems">
+
+    <MSBuild Projects="@(_InnerBuildProjects)"
```

Comment:
This will cause extra msbuild calls (but not extra evaluations as far as I can tell) even for non dotnet5 multi targeting projects. Is it worth doing this extra computation only when it's needed? (dotnet5 and above, or when the TF name requires expanding the platform?)

Reply:
We want to (eventually) support generic TargetFramework aliasing, so we don't want to restrict this to net5.0 and above.  Also, we don't even know if a TargetFramework is net5.0 or above without the parsing that happens in the inner build (which we could probably work around if we didn't want to eventually support aliasing).

Reply:
It's extra evaluations in the not-all-TFs-are-used case, right?

This seems mostly fine. I wonder if it'll cause scheduling/node-locking problems by shifting around the timing for the evaluations/executions/requests. Hopefully not!

Reply:
Evaluation wise it should not be worse than the previous multitargeting scenario, where it also evaluates all innerbuilds but uses one.